docs/adr/20251211-github-issue-management-agreement.md

```diff
@@ -0,0 +1,100 @@
+# Team system for github issue management
+* Status: In Progress
+* Authors: @ifsimicoded
+* Deciders: 
+* As of: 
+
+## Context and Problem Statement
+
+### Current Needs
+- To have a living body of centralized documentation for any project** that can be referenced during the project and after
+- Simplify project progress tracking by PMs, stakeholders and contributors
+- Simplify contributor task tracking (especially for ad-hoc contributors)
+- Simplify onboarding/rebuilding context on projects as contributors hop around from project to project
+- Simplify scanning issues in the Issues view (or any aggregated view) to identify what is relevant to project 
+- Prevent orphan tickets / orphaned work / missed issues
+- Help folks better understand how an issue should be defined. (Creating guidelines could help folks better understand how to breakdown a body of work)
+
+These needs can be accomplished by agreeing to a structure of organizing information, using issue templates and just trying it out to see how it goes knowing it won't be perfect. Hopefully this can be a stepping stone to defining some project processes to facilitate better team alignment on work.
+
+### Is this broadly desired?
+Currently I've seen different individuals put effort into implementing and suggesting (Thank you @vitor) systems for managing github issues and a strong desire to have all project work logged. That leads me to believe we need it, and there's appetite to align on one! So hopefully the buy-in is there.
+
+### Constraints?
+- The Development Seed team currently organizes sprint work within our [Veda Dashboard Team project space](https://github.com/orgs/NASA-IMPACT/projects/17). Github projects allows us to manage issues in a centralized way that are distributed across repos often having different configs for issue meta data (types, labels, etc). That means we cannot rely on repo level meta data fields.
+- Users may already have existing workflows (@aboydnw) that need to be preserved or easily updated.
+- We work on projects across organizations and share repos, so this system will need to also be shared out with and work for our collaborators so there's consistency across our shared repo issues.
+
+## Proposed Guidelines
+
+### Epics 
+(or maybe PI Objectives, I'm not yet sure @aboydnw how you prefer to organize bodies of work*, but I don't think these bodies of work should be tied to PIs as they can often span more than one PI).
+
+All bodies of work should have a parent epic captured in a GH Issue that will be a **living** document for the project for folks to reference both during and after the project work is completed. It should act as the central hub for documentation for the epic that can be referenced in the final epic repo(s) README.
+
+The epic/objective issue should have 
```

Comment:
I would look at the PI Objective template in /veda-architecture. That's been my template for PI Objectives so far. We can update that to include more fields that are useful for the team.

Reply:
Unrelated, should this ADR live in veda-architecture? I'm not sure I understand what goes where -- seems like veda-ui becomes the catch all for things veda, but maybe I misinterpreted, and it should only be focused on library specific items!

Reply:
makes sense, when approved, I'll move this to the wiki, and out of the veda-ui repo